packages/libraries/client/src/internal/utils.ts

```diff
@@ -1,7 +1,48 @@
-import { createHash } from 'node:crypto';
 import { hiveClientSymbol } from '../client.js';
 import type { HiveClient, HivePluginOptions } from './types.js';
 
+export const isCloudflareWorker = Boolean(
+  typeof caches !== 'undefined' && 'default' in caches && caches.default,
+);
+
+async function digest(algo: 'SHA-256' | 'SHA-1', output: 'hex' | 'base64', data: string) {
+  if (typeof crypto !== 'undefined' && typeof TextEncoder !== 'undefined') {
+    const buffer = await crypto.subtle.digest(algo, new TextEncoder().encode(data));
```

Comment:
We have this in `@whatwg-node/fetch`

Reply:
super weird to use WebCrypto API from `whatwg-node/fetch` :))

Reply:
But we already have a dependency here :)
Also some bundlers can complain about the dynamic import below;
https://github.com/kamilkisiela/graphql-hive/pull/4494/files/840154d858b4708d6efb9d7bb1c67ae0ecf6eaa9#diff-1920db0a7d3df2907cd2cc1ed61b516ded3145e21a7487384f60b5d17854d10fR18